.github/workflows/mainnet.yml

```diff
@@ -0,0 +1,24 @@
+name: Mainnet deployment
+
+on:
+  push:
+    branches:
+      -  development
+
+jobs:
+  deploy:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v1
+    - name: Install node
+      uses: actions/setup-node@v1
+      with:
+        node-version: 10
```

Comment:
Should we use 12? In case `now` deprecates 10 soon

Reply:
Sounds good, let's try 12